ministryofjustice/correspondence_tool_staff
# Increase livenessProbe timeoutSeconds to account for CPU heavy processes

## Description
During CPU-heavy job processes, the `livenessProbe` reports a failure and subsequently restarts _erroneous_ pods. This behaviour is most likely occurring when our data-heavy table 'cases' is being backed up by our DB anonymizer task.

This PR increases the `livenessProbe`, and for consistency, the `readinessProbe` timeout value to 15 seconds. 

**_Nb. Pods are being restarted at regular intervals, which could be the root of job-based issues._**

More information can be found in Slack:
https://mojdt.slack.com/archives/C581VE4QZ/p1661863168830619?thread_ts=1661847865.442719&cid=C581VE4QZ 

## Self-review checklist



### Screenshots
**Before:**
![Screenshot 2022-08-30 at 13 28 55](https://user-images.githubusercontent.com/47327051/187447376-96cf795e-3a78-4268-9a5d-90019ef8ac60.png)

**After:**
![Screenshot 2022-08-30 at 13 38 50](https://user-images.githubusercontent.com/47327051/187447460-ef5db5e7-5d78-4fbf-9a82-1a72dc77e94f.png)

### Related JIRA tickets
(https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/864?modal=detail&selectedIssue=CDPT-147&assignee=5c5849024912b735b9e06611)

### Deployment
N/A

### Manual testing instructions
Check pod status and note the READY and RESTARTS columns.

```
kubectl get pods -n track-a-query-qa
```

**_READY_** should not state 0/1 in any row
**_RESTARTS_** should display minimal values with no more than 1 or 2 noted, preferably we want to see 0 in all rows.
